### PR TITLE
[zh] Sync 'scrollintoview' translation from English

### PR DESCRIPTION
**Pull request by starwingcc**
### Description
Update translation of `scrollintoview` api
English page: https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView
Chinese page: https://developer.mozilla.org/zh-CN/docs/Web/API/Element/scrollIntoView

### CODE REVIEW

File: files/zh-cn/web/api/element/scrollintoview/index.md [hunk 1]
```diff
@@ -29,7 +29,10 @@ scrollIntoView(scrollIntoViewOptions)
   - : 一个包含下列属性的对象：
 
     - `behavior` {{optional_inline}}
-      - : 定义动画过渡效果，`auto` 或 `smooth` 之一。默认为 `auto`。
+      - : 定义动画过渡效果，默认为 `auto`。
+        - `smooth`：平滑滚动。
+        - `instant`：立即滚动。
+        - `auto`：滚动行为由{{cssxref("scroll-behavior")}}决定，通常情况下是`instant`。
```
Comment (yin1999) [inline files/zh-cn/web/api/element/scrollintoview/index.md:35]:
```suggestion
      - : 定义滚动是立即的还是平滑的动画。该选项是一个字符串，必须采用以下值之一：
        - `smooth`：滚动应该是平滑的动画。
        - `instant`：滚动应该通过一次跳跃立刻发生。
        - `auto`：滚动行为由 {{cssxref("scroll-behavior")}} 的计算值决定。
```

**Review by yin1999 — COMMENTED** [review]
Please reflect the original en-US text

File: files/zh-cn/web/api/element/scrollintoview/index.md (hunk 1 above)
Comment (starwingChen) [reply files/zh-cn/web/api/element/scrollintoview/index.md]: thanks for your suggestion

**Review by yin1999 — APPROVED** [review]
Thank you @starwingChen. Welcome to the community 🎉

**Pull request merged**